labels/mapping.go

```diff
@@ -10,7 +36,7 @@ var core = []Label{
 	{"2: QA Review", "Requires review by a QA tester", "7cdfe2"},
 	{"3: Reviews Complete", "All reviewers have approved the pull request", "0e8a16"},
 	{"Awaiting Submitter Action", "Blocked on the author", "b60205"},
-	{"Do Not Merge/Awaiting PR", "Awaiting another pull request before merging (e.g. server changes)", "a32735"},
+	// {"Do Not Merge/Awaiting PR", "Awaiting another pull request before merging (e.g. server changes)", "a32735"},
```

Comment:
arguably, no needed for plugins - Do Not Merge + a comment should be enough?

Reply:
We use this label in the server and webapp a lot. 2/5 to have it here for consistency.